app/uk/gov/hmrc/gform/translation/TranslationController.scala

```diff
@@ -16,21 +16,35 @@
 
 package uk.gov.hmrc.gform.translation
 
-import cats.implicits._
-import java.io.{ BufferedOutputStream, ByteArrayInputStream, ByteArrayOutputStream }
-import play.api.libs.json.{ JsObject, Json }
 import akka.stream.scaladsl.StreamConverters
-import play.api.mvc.{ Action, AnyContent, ControllerComponents, Result }
-import scala.concurrent.{ ExecutionContext, Future }
+import cats.implicits._
+import play.api.libs.json.JsObject
+import play.api.libs.json.Json
+import play.api.mvc.Action
+import play.api.mvc.AnyContent
+import play.api.mvc.ControllerComponents
+import play.api.mvc.Result
 import uk.gov.hmrc.gform.controllers.BaseController
-import uk.gov.hmrc.gform.formtemplate.{ FormTemplateService, FormTemplatesControllerRequestHandler }
+import uk.gov.hmrc.gform.core.FOpt
+import uk.gov.hmrc.gform.formtemplate.FormTemplateService
+import uk.gov.hmrc.gform.formtemplate.FormTemplatesControllerRequestHandler
+import uk.gov.hmrc.gform.formtemplate.RequestHandlerAlg
 import uk.gov.hmrc.gform.history.HistoryService
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ FormTemplateId, FormTemplateRaw, FormTemplateRawId }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateId
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateRaw
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateRawId
+
+import java.io.BufferedOutputStream
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
```

Comment:
Why these import changes? This is not consistent with rest of code base?

Reply:
`organize-import` in metals. Do we have a standard? Can we enforce it ?

Reply:
I bet you are alone to use `metals` on this team so it is quite controversial to do this. Can you reconfigure that functionality of not to use it?

Reply:
ok, will do.